x-pack/plugins/security_solution/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_right_panel_overview_tab.cy.ts

```diff
@@ -247,15 +243,11 @@ describe(
         cy.get(DOCUMENT_DETAILS_FLYOUT_INSIGHTS_TAB_ENTITIES_CONTENT).should('be.visible'); // TODO update when we can navigate to Threat Intelligence sub tab directly
       });
 
-      it('should display correlations section', () => {
+      // TODO: skipping this due to flakiness
+      it.skip('should display correlations section', () => {
```

Comment:
after introducing an extra check for case cration toast, this one got flaky. I think part of the reason is that the entries in the list are not constructed in a deterministic order; also it is possible to have an entry appended there twice... should be fixed in a separate PR imho. fyi @PhilippeOberti

Reply:
thanks for raising this, [ticket](https://github.com/elastic/security-team/issues/6864) is created!